akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -37,7 +57,7 @@ import akka.actor.typed.scaladsl.{ ActorContext => SAC }
  */
 @ApiMayChange
 @DoNotInherit
-abstract class Behavior[T] { behavior =>
+abstract class Behavior[T](private[akka] val _tag: Int) { behavior =>
```

Comment:
why named `_tag` instead of `tag`?

Reply:
I think it's better to named `tag` and mark it with `internalUsedOnly`

Reply:
I needed to make the field `private[akka]` to access it in the interpreter and that means it can then not be used as a field/method name in `ExtensibleBehaviors` so I opted for something less likely that a user would want to call a field.

Reply:
ok